localized/en-us/games/eggquiz/eggquizmath_levels.tsv

```diff
@@ -428,27 +427,27 @@ SpecialTest_25	1	250001, 250002, 250003, 250004, 250005
+			260010	word_problem	×æ·UUUUôü Ùð 12 ×ÀçÜØæ¡ ¹ÚèÎè¢Ð ©âÙð ×ÀçÜØô¢ ·UUUè â¢wØæ 4 Åñ¢·UUUUô¢ ×ð¢ ÕÚæÕÚ ·UUUUÚ ÎèÐ ÂýˆØð·UUUU Åñ¢·UUUU ×ð¢ ç·UUUUÌÙè ×ÀçÜØæ¡ ãñ¢?	मार्को ने १२ मछलियाँ खरीदीं। उसने मछलियों की संख्या ४ टैंकों में बराबर कर दी। प्रत्येक टैंक में कितनी मछलियाँ हैं?				3				
 SpecialTest_27	1	270001~270010, 270001~270010, 270001~270010, 270001~270010, 270001~270010												
-			270001	word_problem	×æ§·UUUU ·ðUUU Âæâ 3 ÁôÇ¸è ×ôÁð ãñ¢Ð ÂýˆØð·UUUU ÁôÇ¸è ×ð¢ 2 ×ôÁð ãñ¢Ð ×æ§·UUUU ·ðUUU Âæâ ç·UUUUÌÙð ×ôÁ¸ð ãñ¢?	माइक के पास ३ जोड़ी मोजे हैं। प्रत्येक जोड़ी में २ मोजे हैं। माइक के पास कितने मोज़े हैं?				6				
-			270002	word_problem	°Üè Ùð ¥æ× ·ðUUU 2 Õæòvâ ¹ÚèÎðÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ 4 ¥æ× ãñ¢Ð ©âÙð ç·UUUUÌÙð ¥æ× ¹ÚèÎð?	एली ने आम के २ बॉक्स खरीदे। प्रत्येक बॉक्स में ४ आम हैं। उसने कितने आम खरीदे?				8				
+			270001	word_problem	×æ§·UUUU ·ðUUU Âæâ 3 ÁôÇ¸è ×ôÁð ãñ¢Ð ÂýˆØð·UUUU ÁôÇ¸è ×ð¢ 2 ×ôÁð ãñ¢Ð ×æ§·UUUU ·ðUUU Âæâ ç·UUUUÌÙð ×ôÁ¸ð ãñ¢?	माइक के पास ३ जोड़ी मोजे हैं। प्रत्येक जोड़ी में २ मोजे हैं। माइक के पास कितने मोजे हैं?				6				
+			270002	word_problem	°Üè Ùð ¥æ× ·ðUUU 2 Õæòvâ ¹ÚèÎðÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ 4 ¥æ× ãñ¢Ð ©âÙð ç·UUUUÌÙð ¥æ× ¹ÚèÎð?	एली ने आम के २ बोक्स खरीदे। प्रत्येक बोक्स में ४ आम हैं। उसने कितने आम खरीदे?				8				
 			270003	word_problem	¿Úæ»æã ×ð¢ 3 ÖðÇ¸ð¢ ãñ¢Ð ÂýˆØð·UUUU ÖðÇ¸ ·ðUUU 4 Õ‘¿ð ãôÌð ãñ¢Ð ¿Úæ»æã ×ð¢ ç·UUUUÌÙð ×ð×Ùð ãñ¢?	चरागाह में ३ भेड़ें हैं। प्रत्येक भेड़ के ४ बच्चे होते हैं। चरागाह में कितने मेमने हैं?				12				
-			270004	word_problem	ÁðÙ Ùð xÜæòßÚ ÂæòÅ ×ð¢ 3 ÈêUUUÜô¢ ·UUUè ç»ÙÌè ·UUUèÐ ÂýˆØð·UUUU ÈêUUUÜ ×ð¢ 5 Â¢¹éÇ¸çØæ¡ ãôÌè ãñ¢Ð âÖè ×ð¢ ç·UUUUÌÙè Â¢¹éÇ¸çØæ¡ ãñ¢?	जेन ने 3लॉवर पॉट में ३ फूलों की गिनती की। प्रत्येक फूल में ५ पंखुड़यिाँ होती हैं। सभी में कितनी पंखुड़यिाँ हैं?				15				
+			270004	word_problem	ÁðÙ Ùð xÜæòßÚ ÂæòÅ ×ð¢ 3 ÈêUUUÜô¢ ·UUUè ç»ÙÌè ·UUUèÐ ÂýˆØð·UUUU ÈêUUUÜ ×ð¢ 5 Â¢¹éÇ¸çØæ¡ ãôÌè ãñ¢Ð âÖè ×ð¢ ç·UUUUÌÙè Â¢¹éÇ¸çØæ¡ ãñ¢?	जेन ने फ्लावर पॉट में ३ फूलों की गिनती की। प्रत्येक फूल में ५ पंखुड़यिाँ होती हैं। सभी में कितनी पंखुड़यिाँ हैं?				15				
 			270005	word_problem	¹ðÌ ÂÚ 2 ßñ»Ù ãñ¢Ð ÂýˆØð·UUUU ßñ»Ù ×ð¢ 8 Õ‘¿ð ãôÌð ãñ¢Ð ç·UUUUÌÙð Õ‘¿ð ßñ»Ùô¢ ×ð¢ ãñ¢?	खेत पर २ वैगन हैं। प्रत्येक वैगन में ८ बच्चे होते हैं। कितने बच्चे वैगनों में हैं?				16				
 			270006	word_problem	°·UUUU ×Ïé×v¹è ·ðUUU 6 ÂñÚ ãôÌð ãñ¢Ð 2 ×Ïé×çv¹Øô¢ ·ðUUU ç·UUUUÌÙð ÂñÚ ãôÌð ãñ¢?	एक मधुमक्खी के ६ पैर होते हैं। २ मधुमक्खियों के कितने पैर होते हैं?				12				
-			270007	word_problem	°·UUUU Õ}æ¹ ·ðUUU 2 ÂñÚ ãôÌð ãñ¢Ð 3 ÕÌ¹ ·ðUUU ç·UUUUÌÙð ÂñÚ ãñ¢?	एक बतख के २ पैर होते हैं। ३ बतख के कितने पैर हैं?				6				
-			270008	word_problem	àæðËÜè Ùð ÇôÙÅ÷â ·ðUUU 4 Õæòvâ ¹ÚèÎðÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ 5 ÇôÙÅ÷â ãôÌð ãñ¢Ð ©âÙð ·éUUUÜ ç·UUUUÌÙð ÇôÙÅ÷â ¹ÚèÎð?	शेल्ली ने डोनट्स के ४ बॉक्स खरीदे। प्रत्येक बॉक्स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?				20				
+			270007	word_problem	°·¤ Õžæ¹ ·Ô¤ 2 ÂñÚU ãôÌð ãñ´Ð 3 Õžæ¹ ·Ô¤ ç·¤ÌÙð ÂñÚU ãñ´?	एक बतख के २ पैर होते हैं। ३ बतख के कितने पैर हैं?				6				
```

Comment:
Chanakya text says:
एक बाख के 2 पैर होते हैं। 3 बाख के कितने पैर हैं?

Fix it

Reply:
It is right, was it showing in the app?

Reply:
![image](https://user-images.githubusercontent.com/34193530/61103914-acedf580-a491-11e9-9811-eafc17d570df.png)

Reply:
Use this [link](https://www.pramukhfontconverter.com/hindi/chanakya)

Reply:
Resolved